pipeline/misc/summary_stats_merger.ipynb

```diff
@@ -285,7 +285,7 @@
     "            except:\n",
     "                raise ValueError(f'According to config_file, input summary statistics should have the following columns: %s' % list(config.values()))\n",
     "            sumstats.columns = list(config.keys())\n",
-    "        sumstats.SNP = 'chr'+sumstats.CHR.astype(str) + ':' + sumstats.POS.astype(str) + '_' + sumstats.A0.astype(str) + '_' + sumstats.A1.astype(str)\n",
+    "        sumstats.SNP = sumstats.index + '_' + sumstats.A0.astype(str) + '_' + sumstats.A1.astype(str)\n",
```

Comment:
I am using ID  as index, such as `chr:pos`.

Reply:
@changebio Here ID is variant_id as we have established. Then, as I commented before, "if so, they better have a0 and a1 already before you make them index. Otherwise if you use `chr:pos`, there will be duplicates. If in variant ID there are already a0 and a1 then what's the point of adding them here?", right? Please address to my concerns. Thanks!

Reply:
There are two important function in these pipeline: snps_match and pair_match. pair_match is the same as the R function allele.qc, which deal with strand flip and reverse for `paired SNPs`. 
How to define `paired SNPs` is very important. If we assume there are no duplicated chr:pos, we can use chr:pos to define SNP pairs, the potential SNPs which could be the same, but with strand flip and reverse.
snps_match function is to find and match paired SNPs in two different files.
Here the `ID` is to make the index for SNPs which can be map in different files, so we know they are `paired SNPs`.
If there are duplicated `chr:pos`, like Hao's case, we can use `chr:pos:gene` as index. So SNPs with the same `chr:pos:gene` are paired SNPs.